sympy/integrals/tests/test_heurisch.py

```diff
@@ -265,6 +260,28 @@ def test_pmint_besselj():
 
     assert heurisch(f, x) == g
 
+    f = jn(nu + 1, x)/jn(nu, x)
+    g = nu*log(x) - log(jn(nu, x))
+
+    assert heurisch(f, x) == g
+
+@slow
+def test_pmint_bessel_products():
+    # Note: Derivatives of Bessel functions have many forms.
+    # Recurrence relations are needed for comparisons.
+    if ON_TRAVIS:
+        skip("Too slow for travis.")
```

Comment:
Too bad the tests are too slow. They seem to be passing for me on my machine, though.